global-script.lua

```diff
@@ -3088,4 +3148,87 @@ function tCompare(t1,t2)
     return table.concat(newTab,"|")
     end
     if cc2(t1) == cc2(t2) then return true else return false end
-end
\ No newline at end of file
+end
+
+
+function swapPlayerAreas(a, b)
+    local function tableSwap(table)
+        local temp = table[a]
+        table[a] = table[b]
+        table[b] = temp
+    end
+    local function tintSwap(table)
+        local oa = table[a]
+        local ob = table[b]
+        if type(oa) == "string" then
```

Comment:
when would this be a string? onLoad always converts from guid to object right?

Reply:
`elementScanZones` is the exception here, though as it turns out the current implementation no longer moves them and just tableswaps them instead so this exception should not be needed.

Reply:
Ah, `readyTokens` is also an exception here.